phpBB/index.php

```diff
@@ -161,8 +161,7 @@
 	$db->sql_freeresult($result);
 }
 
-// Assign index specific vars
-$template->assign_vars(array(
+$board_statistics = array(
```

Comment:
This is not just the board statistics but rather most of the index template data apart from the forums. Maybe just use a variable name like $template_data